modules/core/include/opencv2/core/hal/intrin.hpp

```diff
@@ -316,7 +318,7 @@ template<typename _Tp> struct V_RegTraits
     CV_INTRIN_DEFINE_WIDE_INTRIN_ALL_TYPES(v256)
     CV_INTRIN_DEFINE_WIDE_INTRIN(double, v_float64, f64, v256, load)
     inline void vx_cleanup() { v256_cleanup(); }
-#elif CV_SIMD128
```

Comment:
IMO, we only need to fallback into dummy types and dummy intrinsics just for template programming and also dispatching simd128 intrin tests. 
However we need this for now to avoid compiling errors for non-simd optimizations.